Library/ViewModels/MessageBannerViewModel.swift

```diff
@@ -53,14 +54,34 @@ MessageBannerViewModelInputs, MessageBannerViewModelOutputs {
     self.messageTextAlignment = bannerType.map { $0.textAlignment }
 
     self.bannerMessage = self.messageBannerConfiguration.signal.skipNil().map(second)
+    self.bannerMessageAccessibilityLabel = self.bannerMessage
+      // todo: Needs to be localized - Strings.banner_tap_to_dismiss
+      .map { "\($0), double tap to dismiss" }
 
     let bannerViewShouldHide = self.showBannerViewProperty.signal.negate()
 
-    let postAnimationBannerViewShouldHide = self.bannerViewIsHiddenProperty.signal
-      .filter { isFalse($0) }
+    let dismissBanner = self.bannerViewIsHiddenProperty.signal
+      .filter(isFalse)
+
+    // Dismisses the banner after 4 seconds when VoiceOver is OFF
```

Comment:
Were we certain about the use of `debounce` here vs `delay`? I remember chatting about it but can't recall 😅

Reply:
Yes, we actually really want to use `debounce`. The banner was showing a weird behaviour when using `delay`.